### PR TITLE
Docker container

### PR DESCRIPTION
**Pull request by RAFAY-AAMIR-GULL**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    wget \
+    unzip
+
+# DOWNLOAD AND INSTALL PYTHON AND PIP
+RUN apt update &&\
+    apt install -y \
+    curl \
+    python3-pip
+
+RUN pip3 --default-timeout=1000 install torchvision tensorboardx
```
Comment (Hassannawazish) [inline Dockerfile:28]: Need to add more dependencies and in addition we can make a requirement.txt to list down all packages and install that file.

File: .devcontainer/devcontainer.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+		"--net=host",
+		"--gpus",
+		"all",
+		"-e",
+		"DISPLAY=${env:DISPLAY}",
+		"-v",
+		"/tmp/.X11-unix:/tmp/.X11-unix",
+		"--privileged",
+		"-v",
+		"/media/rafay/Windows-SSD/other/data/open_source/3D_retinaNet/:/workspace/dataset"
```
Comment (Hassannawazish) [inline .devcontainer/devcontainer.json:39]: Are you sure about the dependencies? because there is hardcoded path.

**Review by Hassannawazish — COMMENTED** [review]
The rest of work is fine and I have created the image and tested on my side it is working fine apart from some package installations.

**Review by Hassannawazish — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (SomaiaElbaradey) [inline Dockerfile:28]: needs to add TODO comment to make sure of that.

**Review by SomaiaElbaradey — APPROVED** [review]

**Review by Sameed66 — APPROVED** [review]
After the above mentioned reviews, docker configuration looks fine from my side

**Pull request merged**